src/index.ts

```diff
@@ -1,3 +1,4 @@
+import './global';
```

Comment:
I think something like

```suggestion
/// <reference types="./global" />
```

Might also work?

Reply:
Actually, looks like we don't need it at all? Maybe because it is part of the `include`?